Feature

WestonF opened the pull request.

WestonF committed:
CA initial functional commit

WestonF committed:
removed CA, added server

WestonF committed:
Remove some non-publish files

calvindo95 commented on utpsrv.cpp:
```diff
@@ -0,0 +1,247 @@
+const int SMALL_CONSTANT = 2;
+
+
+class config_keeper
```
It seems like you're not using this class anywhere or maybe you just haven't gotten to actually using it yet. I could be wrong though

calvindo95 commented on log.txt:
```diff
@@ -0,0 +1 @@
+--PERFORMANCE LOG START --
```
The log file should be added to the .gitignore, as we don't need it or its contents in the repo

calvindo95 commented:
You should probably consider moving the classes into separate files to make it easier to understand and read what each class does. And it declutters your 'main' file.

WestonF committed:
Added file response

WestonF committed:
Added responses and header files

Added PUT, POST, and DELETE responses
Moved some classes to header files

WestonF committed:
Mass Update

update to get github on track with current project progress, from which I can begin to correctly manage regular commits and reviews
